packages/datetime/src/_datepicker.scss

```diff
@@ -17,13 +17,24 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
   line-height: 1;
 }
 
+.#{$ns}-divider {
```

Comment:
TODO: gonna pull this out to its own component in a separate PR, in progress now.